tests/conftest.py

```diff
@@ -52,6 +53,27 @@ def eager_task_loop_factory() -> asyncio.AbstractEventLoop:
     )
 
 
+@pytest.fixture(autouse=True)
+def blockbuster() -> Iterator[BlockBuster]:
+    with blockbuster_ctx(
+        "anyio", excluded_modules=["anyio.pytest_plugin", "anyio._backends._asyncio"]
+    ) as bb:
+        bb.functions["os.stat"].can_block_in("anyio/streams/tls.py", "wrap")
+        bb.functions["socket.socket.accept"].can_block_in(
+            "anyio/_core/_asyncio_selector_thread.py", {"get_selector"}
```

Comment:
`get_selector` initiates a `Selector()` which creates a socket pair.
Windows doesn't have a `socketpair` syscall so it it emulated by `socket._fallback_socketpair`.
[_fallback_socketpair](https://github.com/python/cpython/blob/86dab0e67e7e75c4c77e994b99a842f033ddbe46/Lib/socket.py#L598) does a `listen` on a server socket, creates a non-blocking client socket and connects to the server, then calls a blocking `accept` on the server socket. The accept wait time should be very short since the client socket is connecting but I'm not sure how long it is or if it's a problem.
Also the `Selector` created by `get_selector` is cached.
So maybe it can be kept as-is ?

Reply:
I don't see this as a problem, so let's just keep this exemption.